src/services/routes/org.routes.spec.ts

```diff
@@ -186,7 +188,147 @@ describe('org routes', () => {
       });
 
       expect(response).toHaveProperty('statusCode', HTTP_STATUS_CODES.BAD_REQUEST);
-      expect(response.json()).toHaveProperty('type', CreationProblemType.MALFORMED_AWALA_ENDPOINT);
+      expect(response.json()).toHaveProperty('type', OrgProblemType.MALFORMED_AWALA_ENDPOINT);
+    });
+  });
+
+  describe('update', () => {
+    const injectionOptions: InjectOptions = {
+      method: 'PATCH',
+      url: `/orgs/${ORG_NAME}`,
+    };
+    const getOrgSuccessResponse = {
+      didSucceed: true,
+
+      result: {
+        name: ORG_NAME,
+        memberAccessType: 'INVITE_ONLY',
+      },
+    } as const;
+
+    test('Empty parameters should be accepted', async () => {
+      const serverInstance = await makeServer();
+      const payload: OrgSchemaPatch = {};
+      mockGetOrg.mockResolvedValueOnce(getOrgSuccessResponse);
+      mockUpdateOrg.mockResolvedValueOnce({
+        didSucceed: true,
+      });
+
+      const response = await serverInstance.inject({
+        ...injectionOptions,
+        payload,
+      });
+
+      expect(response).toHaveProperty('statusCode', 200);
+    });
+
+    test('name matching the url parameter should be accepted', async () => {
```

Comment:
Nitpick: The convention is to title-case these strings, so "n" in "name" should be uppercase.

This also applies to other tests in this file.